packages/web/src/components/__tests__/Resource.test.tsx

```diff
@@ -36,7 +36,7 @@ jest.mock("../../App.styles", () => ({
   },
 }));
 
-const mockUseResource = useResource as jest.Mock;
+const mockUseResource = useResources as jest.Mock;
```

Comment:
```suggestion
const mockUseResources = useResources as jest.Mock;
```

Looks like the return in this test on line 47 will need updated too:

```js
it("renders a loading spinner when a resource is loading", () => {
    mockUseResources.mockReturnValueOnce({ status: "loading" });
```

Reply:
c66df1b0bf9d707e20650551321a69d59884f473